### ISSUE TITLE
请问如何使用内置表达式计算自定义DataFrame的衍生指标

### ISSUE THREAD

**Issue by oldweima** [microsoft/qlib#1452]
## ❓ Questions and Help

我从数据库读取了df，希望使用内置表达式计算先生特征，再使用StaticDataLoader读入。
但不清楚如何使用内置表达式计算自定义DataFrame的衍生指标。

> df_mongo['future_10_max'] = df_mongo.iloc[::-1].rolling(2)['$close'].max()       
print(df_mongo)
'''
feature                                                                                    label
                                      $close     $open     $high      $low      future_10_max
datetime   instrument                                                                                                          
1990-12-19 000001        99.980    96.050    99.980    95.790  104.390
'''

> # df_mongo['close_ma'] = EMA('$close', 10)                   # 会报错

> # labels = ['Ref($close, -2)/Ref($close, -1) - 1']

> sdl_df = StaticDataLoader(config=df_mongo)

**Comment by Fivele-Li** [comment]
Hi, you can try the methods mentioned in issue https://github.com/microsoft/qlib/issues/1501 and see if they can solve your problem.
